x-pack/plugins/observability_solution/dataset_quality/server/services/index.ts

```diff
@@ -8,3 +8,4 @@
 export { dataStreamService } from './data_stream';
 export { indexStatsService } from './index_stats';
 export { datasetQualityPrivileges } from './privileges';
+export { DataTelemetryService } from './data_telemetry/data_telemetry_service';
```

Comment:
> [!WARNING]
> Is there any usage that requires exporting this service from the plugin entry point? If not, we can keep it encapsulated in the plugin.

Reply:
There's no current usage that the service is consumed outside of the plugin. But is exporting from `/server/services/index.ts` is exporting from plugin's entry point?

Reply:
My bad, didn't notice the wording `services` in the middle, resolved!